pyproject.toml

```diff
@@ -13,15 +13,15 @@ python = "^3.7.1"
 thrift = "^0.16.0"
 pandas = "^1.3.0"
 pyarrow = [
-    {version = ">=9.0.0", python = ">=3.7,<3.11"},
+    {version = ">=6.0.0", python = ">=3.7,<3.11"},
     {version = ">=10.0.1", python = ">=3.11"}
 ]
 lz4 = "^4.0.2"
 requests="^2.18.1"
 oauthlib="^3.1.0"
 numpy = [
-    {version = "1.21.1", python = ">=3.7,<3.8"},
-    {version = "1.23.4", python = ">=3.8"}
+    {version = ">=1.16.6", python = ">=3.7,<3.8"},
```

Comment:
It looks like `numpy` is barely used in this library so I think pretty much any version should work. I chose `>=1.16.6` because that is what the latest version of `pyarrow` requires.